server/monitor_test.go

```diff
@@ -1313,18 +1313,18 @@ func TestConcurrentMonitoring(t *testing.T) {
 			for i := 0; i < 150; i++ {
 				resp, err := http.Get(url + endpoint)
 				if err != nil {
-					t.Fatalf("Expected no error: Got %v\n", err)
```

Comment:
@kozlovic ok I will close this one since you had ideas on the fix already, thanks!